src/Native/pkg/runtime.native.System.IO.Compression/win10/runtime.native.System.IO.Compression.pkgproj

```diff
@@ -8,9 +8,8 @@
   </PropertyGroup>
 
   <ItemGroup>
-    <!-- use Win10 temporarily as the distinguishing factor, in reality the distinction is app-container vs not. -->
     <File Include="$(WinNativePath)..\..\NetCoreForCoreCLR\native\clrcompression.dll">
-      <TargetPath>runtimes/win10-$(PackagePlatform)/native</TargetPath>
+      <TargetPath>runtimes/win10-$(PackagePlatform)/lib/netcore50</TargetPath>
```

Comment:
That's what I was thinking but the discussion in that issue led me to believe otherwise. I don't know how we would test it.